package.json

```diff
@@ -3,10 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "test": "jest",
-    "report": "node tools/reporter",
-    "validate": "node tools/vuln_valid",
-    "create-npm-index": "node tools/create_index/create_npm_index.js",
-    "create-core-index": "node tools/create_index/create_core_index.js"
+    "report": "node tools/reporter"
```

Comment:
What did we decide eventually? we're moving out all the Pull Requests to be opened in the security-advisories repo too =, right? meaning we need to move this tool out to the other repo as well.

Reply:
FYI that it needs to move there with axios, inquirer, jest and yargs probably.